styles/index.css

```diff
@@ -0,0 +1,355 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  color: #555;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  padding: 110px 40px;
+  display: block;
+  position: relative;
+  margin-top: -50px;
+  color: #555;
+  z-index: -1;
+}
+
+.header::after {
+  content: '';
+  background: url('../assets/bg-home.jpg');
+  background-size: cover;
+  opacity: 0.5;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  position: absolute;
+  z-index: -1;
+}
+
+.header__header2 {
+  color: #ec5242;
+}
+
+.header__header1 {
+  font-family: 'COCOGOOSE', sans-serif;
+  padding: 20px 0;
+  font-size: 1.9em;
+  font-weight: 700;
+  color: #ec5242;
+}
+
+.header__text {
+  padding: 20px;
+  border: 3.5px solid #fff;
+  background-color: #f7f7f7;
+  margin: 5px;
+  line-height: 25px;
+}
+
+.header__date {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-size: 1.3em;
+  padding: 20px 0 5px 0;
+}
+
+.main {
+  background-image: url('../assets/main-bg.png');
+  background-color: #272a31;
+  color: #fff;
+  padding: 40px 0;
+}
+
+.main__line {
+  width: 15%;
+  height: 2px;
+  background-color: #ec5242;
+  margin-top: 15px;
+}
+
+.main__heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main__List {
+  padding-top: 30px;
+}
+
+.main__List__item {
+  display: grid;
+  grid-template-columns: 18% 18% 58%;
+  background-color: #5f5f5f50;
+  grid-column-gap: 3%;
+  padding: 4% 1%;
+  margin: 20px;
+}
+
+.main__List__item__SubItem {
+  align-self: center;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 20px;
+}
+
+.main__List__item__title {
+  font-size: 17px;
+  color: #ec5242;
+  font-weight: bold;
+}
+
+.btn-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 40px;
+}
+
+.main-btn {
+  padding: 30px 15%;
+  background-color: #ec5242;
+  color: #fff;
+  border-style: none;
+  font-size: 20px;
+}
+
+.speakers {
+  padding: 20px;
+}
+
+.speakers__line {
+  width: 15%;
+  height: 2px;
+  background-color: #ec5242;
+  margin-top: 15px;
+}
+
+.speakers__heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.speakers__list__item {
+  display: grid;
+  grid-template-columns: 35% 57%;
+  grid-column-gap: 8%;
+  margin-top: 40px;
+  box-sizing: border-box;
+}
+
+.speaker__list__item__img {
+  background-image: url('../assets/chess.jpg');
```

Comment:
- [ ] To make your project look more professional, you should consider cropping that black banner from the bottom of your image. 
- [ ] [OPTIONAL]: You should also consider adding a `filter` to your image, to make it grey, in order to match the original design better.